core/domain/state_domain.py

```diff
@@ -503,9 +685,12 @@ def get_all_html_content_strings(self):
         if self.id in (
                 'ItemSelectionInput', 'MultipleChoiceInput',
                 'DragAndDropSortInput'):
-            customization_args_html_list = (
-                self.customization_args['choices']['value'])
-            html_list = html_list + customization_args_html_list
+            try:
```

Comment:
Why are we doing this?

Reply:
I have done it because while doing migration there is getting a key error of ``` choices ``` while extracting HTML from the state.
so to avoid key error I do it.

Reply:
Please do not use try and except in places where you aren't sure what's going on! Please check what's going on here and remove try and except.

Reply:
Please go through this comment!

Reply:
Any updates here?

Reply:
yes, it will be cover in my upcoming commit.
Currently fixing all tests.
And next, I will do this, since this needs more investigation i think.